inspirehep/modules/workflows/workflows/author.py

```diff
@@ -88,17 +87,6 @@
 ]
 
 
-SEND_UPDATE_NOTIFICATION = [
-    do_not_repeat('create_ticket_author_submission_curator_update')(
-        create_ticket(
-            template="authors/tickets/curator_update.html",
```

Comment:
Is the template still in use? If not, maybe you can delete it?

Reply:
It's not used- removed it

Reply:
It's not used- removed it